<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyListItem</class>
 <widget class="QWidget" name="MyListItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>20</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>classname</string>
   </property>
  </widget>
  <widget class="QLabel" name="teacherLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>teacher</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeAndPositionLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>time_and_position</string>
   </property>
  </widget>
  <widget class="QLabel" name="creditLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>credit</string>
   </property>
  </widget>
  <widget class="QLabel" name="ratioLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ratio</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
